Name  `_error`  not in scope

I have a name error when the `tracing` feature is enabled. `_error` is indeed not in scope.

https://github.com/mondeja/leptos-fluent/blob/57936de57bb2ae04892c9e0b82768519f36f9b26/leptos-fluent/src/http_header.rs#L21 

Version: leptos-fluent v0.2.10

```rust
  Compiling leptos-fluent v0.2.20
error[E0425]: cannot find value `_error` in this scope
  --> /Users/.../.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/leptos-fluent-0.2.20/src/http_header.rs:21:29
   |
21 | ...                   _error
   |                       ^^^^^^ not found in this scope
```

--- a/leptos-fluent/src/http_header.rs
+++ b/leptos-fluent/src/http_header.rs
@@ -12,16 +12,17 @@ pub fn parse(header: &str) -> Vec<String> {
             let mut quality = 1.0_f32;
             for part in parts {
                 if let Some(value) = part.trim().strip_prefix("q=") {
-                    quality = value.parse().unwrap_or({
-                        #[cfg(feature = "tracing")]
+                    let result = value.parse();
+                    #[cfg(feature = "tracing")]
+                    let result = result.inspect_err(|error|{
                         tracing::trace!(
                             "Invalid quality value {:?} in Accept-Language header \"{}\": {:?}",
                             value,
                             header,
-                            _error
+                            error
                         );
-                        1.0
                     });
+                    quality = result.unwrap_or(1.0);
                 }
             }
 

